themes/default/content/blog/replicating-data-across-multi-region-apps/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Replicating Data Across Multi Region Apps"
+date: 2023-12-15T11:03:10Z
+draft: false
+meta_desc: Learn about the tradeoffs in distributed databases when building for multiple regions.
+meta_image: meta.png
```

Comment:
Adding a reminder here that we can't publish this until you have a meta image for the post.